apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -1019,6 +1019,7 @@
   };
 
   MockNavigatormozMobileMessage.send = function(number, text, success, error) {
+    var now = Date.now();
```

Comment:
this is needed too, as otherwise we always reuse the same "now". This was already done in sendMMS below.